src/main/java/com/github/underscore/U.java

```diff
@@ -2512,6 +2512,28 @@ public boolean test(final Map<K, V> item) {
         };
     }
 
+    public static String[] explode(final String input) {
+        if (isNull(input)) {
+            return new String[0];
+        }
+        String[] result = new String[input.length()];
```

Comment:
It may be a list of String not array. It may be moved to lodash package. We have some string methods in it.